### PR TITLE
Menu content / App.css created

### PR DESCRIPTION
**Pull request by HunterMarion**
Basic Menu content / App.css created

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,47 @@
...
+}
+
+body {
+  background-color: #EBEDF2;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+h1 {
```
Comment (michellenicolephillips) [inline src/App.css]: Unless we plan on having the h1 be styled differently, you can combine header styles into one like so:
`h1, h2, h3, h4 {
  font-family: Garamond, sans-serif;
  color: #591902;
}`
Also, can you include down to h4? For now, the catering page is using up to h4. Might change later.

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,47 @@
...
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #591902;
+}
+
+nav {
+  font: Arial, Helvetica, sans-serif;
+  color: #591902;
+}
+
+body {
```
Comment (michellenicolephillips) [inline src/App.css:13]: You can include the main font we're using in the body tag so it doesn't have to be defined in every element tag we use it in

**Review by michellenicolephillips — COMMENTED** [review]
Everything looks good! There's a couple of comments on combining css stylings so the file doesn't have to be as long but it doesn't break anything if they're separated out how they are now

**Comment by HunterMarion** [comment]
I added your changes and simplified the CSS tags.

**Review by michellenicolephillips — APPROVED** [review]

**Pull request merged**